src/e2e/pages/welcomeScanPage.ts

```diff
@@ -67,4 +68,28 @@ export class WelcomePage {
     this.modalConfirmButton = page.getByRole("button", { name: "Confirm" });
     this.modalEditButton = page.getByRole("button", { name: "Edit" });
   }
+
+  async goThroughFirstScan() {
+    // confirm get started step elements
+    expect(await this.getStartedStep.count()).toEqual(3);
+    await expect(this.page.getByText("Get started")).toBeVisible();
+    await expect(this.page.getByText("Enter info")).toBeVisible();
+    await expect(this.page.getByText("Find exposures")).toBeVisible();
```

Comment:
Not sure if these locators should be in a POM, but we can also file a followup ticket to refactor in "the future" and land this for now.